subsys/bluetooth/controller/Kconfig

```diff
@@ -289,6 +289,14 @@ config BT_CTLR_TX_PWR_MINUS_40
 
 endchoice
 
+config BT_CTRL_TX_PWR_DYNAMIC_CTRL
```

Comment:
Rename this to `BT_CTLR_TX_PWR_DYNAMIC_CONTROL` as is more verbose and avoids the `CTRL` vs `CTLR` confusion.

`BT_CTLR` is the default Zephyr Bluetooth controller Kconfig abbreviation.

Reply:
Solved [here](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/b8c7a15362e46dbcc1238e7bf79694779119a062#diff-fdf68fcfb91aad56ca9d8b0a627c1ab4R292)